pkg/release/uninstall.go

```diff
@@ -0,0 +1,81 @@
+package release
+
+import (
+	"errors"
+	"github.com/helmwave/helmwave/pkg/feature"
+	"github.com/helmwave/helmwave/pkg/helper"
+	"github.com/helmwave/helmwave/pkg/parallel"
+	log "github.com/sirupsen/logrus"
+	"helm.sh/helm/v3/pkg/action"
+	helm "helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+func (rel *Config) Uninstall() (*release.UninstallReleaseResponse, error) {
+	cfg, err := helper.ActionCfg(rel.Options.Namespace, helm.New())
+	if err != nil {
+		return nil, err
+	}
+
+	client := action.NewUninstall(cfg)
+
+	return client.Run(rel.Name)
+
+}
+
+func Uninstall(releases []*Config) (err error) {
+	if len(releases) == 0 {
+		return emptyReleases
+	}
+
+	log.Info("🛥 Sync releases")
+	var fails []*Config
+
+	if feature.Parallel {
+		wg := parallel.NewWaitGroup()
+		log.Debug("🐞 Run in parallel mode")
+		wg.Add(len(releases))
+		for i := range releases {
+			go func(wg *parallel.WaitGroup, release *Config, fails *[]*Config) {
```

Comment:
Slices are already passed by reference (as they are references themselves), so getting its reference is pretty useless